consensus/types/src/light_client_bootstrap.rs

```diff
@@ -37,13 +41,31 @@ impl<T: EthSpec> LightClientBootstrap<T> {
         let current_sync_committee_branch =
             beacon_state.compute_merkle_proof(CURRENT_SYNC_COMMITTEE_INDEX)?;
         Ok(LightClientBootstrap {
-            header,
+            header: header.into(),
             current_sync_committee: beacon_state.current_sync_committee()?.clone(),
             current_sync_committee_branch: FixedVector::new(current_sync_committee_branch)?,
         })
     }
 }
 
+impl<T: EthSpec> ForkVersionDeserialize for LightClientBootstrap<T> {
+    fn deserialize_by_fork<'de, D: Deserializer<'de>>(
+        value: Value,
+        fork_name: ForkName,
+    ) -> Result<Self, D::Error> {
+        match fork_name {
+            ForkName::Altair => Ok(serde_json::from_value::<LightClientBootstrap<T>>(value)
+                .map_err(serde::de::Error::custom))?,
+            ForkName::Base | ForkName::Merge | ForkName::Capella | ForkName::Deneb => {
```

Comment:
bellatrix variant is equal to Altair variant so should be supported. The header was extended on capella

https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/light-client/fork.md#upgrading-light-client-data

Reply:
Nice catch! Fixed in https://github.com/sigp/lighthouse/pull/3954/commits/d23bba7ce06902e96764290a7eb6c0464362bd44